app/controllers/authenticate_controller.rb

```diff
@@ -119,6 +122,17 @@ def authenticator_input
     )
   end
 
+  def render_authn_token(authn_token)
```

Comment:
```suggestion
  def rendered_authn_token(authn_token)
```

the method name should match the returned object

Reply:
there is no returned object, the function renders the authn token, "render_authn_token" explains the action that will happen

Reply:
pretty sure that `render` returns an object but i won't block the PR for this because it also makes sense to match the `render` call below.